templates/CRM/Admin/Page/MosaicoTemplates.tpl

```diff
@@ -66,6 +66,54 @@
           }); 
         }
       });
+      //Copy mosaico template
+      $('.copy-template').click(function(event) {
+	$el = $(this);
+	var msgTplId = $(this).attr('value');
+	console.log('copying msg template ' + msgTplId);
+	var postUrl = {/literal}"{crmURL p='civicrm/mosaico/ajax/copy' h=0 }"{literal};
+	console.log('posturl copy=' + postUrl);
+	$.ajax({ type: "POST", url: postUrl, data: {id:msgTplId}, async: true, dataType: 'json',
+	  success: function(result) {
+	    console.log(result.newMosaicoTplId);
+	    console.log(result.hash_key);
+	    console.log(result.name);
+	    //create mos template and update meta data in civicrm_mosaico_msg_template table
+	    createmostpl(result.newMosaicoTplId, result.hash_key, result.name);
+	    CRM.refreshParent($el);
```

Comment:
look at how builder redirects to mosaic tab. May be that part of JS needs to be activated here